speech/snippets/transcribe_chirp_auto_detect_language.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+# [START speech_transcribe_chirp_auto_detect_language]
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.speech_v2 import SpeechClient
+from google.cloud.speech_v2.types import cloud_speech
+
+
+def transcribe_chirp_auto_detect_language(
+    project_id: str,
+    audio_file: str,
+    region: str = "us-central1",
+) -> cloud_speech.RecognizeResponse:
+    """Transcribe an audio file and auto-detect spoken language using Chirp."""
```

Comment:
Would be nice to include info on requirements for the audio in the docstring. What format, what bitrate, max length etc.

Reply:
Done